README.md

```
@@ -1,27 +1,35 @@
-*   [Commands Overview](#commands-overview)
-*   [Environment Setup](#environment-setup)
-*   [Create Page Command](#create-page-command)
-*   [Upload Attachment Command](#upload-attachment-command)
-*   [Update Page Command](#update-page-command)
-*   [Complete Usage Examples](#complete-usage-examples)
-*   [API Reference](#api-reference)
-*   [Troubleshooting](#troubleshooting)
-*   [Notes and Best Practices](#notes-and-best-practices)
+- [Quick Start](#quick-start)
+- [Commands Overview](#commands-overview)
+- [Environment Setup](#environment-setup)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)